render-pokemon.js

```diff
@@ -6,16 +9,56 @@ export function renderPokemon(pokemon) {
     const price = document.createElement('h2');
     const category = document.createElement('h3');
     const button = document.createElement('button');
+    const input = document.createElement('input');
+
     name.textContent = pokemon.name;
     img.src = pokemon.image;
-    img.style.height = '250px';
-    img.style.width = '250px';
+    img.style.height = '200px';
+    img.style.width = '200px';
     description.textContent = pokemon.description;
-    price.textContent = pokemon.price;
+    price.textContent = '$' + pokemon.price.toFixed(2);
     category.textContent = pokemon.category;
     list.style.listStyle = 'none';
     button.value = pokemon.id;
     button.textContent = 'Add to Cart';
-    list.append(name, img, description, price, category, button);
+    input.type = 'number';
+    input.min = '1';
+    input.max = '99';
+    input.value = 1;
+    
+    button.addEventListener('click', () => {
+        let cart = localStorage.getItem('CART');
+
+        if (!cart) {
+            cart = [];
```

Comment:
You've got some extra white space scattered about in this file that makes your code a little hard to read.